buildroot-external/patches/occu/0135-WebUI-Add-ControlPanel-AdvancedSettings/occu/WebUI/www/config/cp_advancedsettings.cgi

```diff
@@ -374,6 +381,22 @@ proc action_put_page {} {
     puts "var url = \"$env(SCRIPT_NAME)?sid=\" + SessionId;"
     puts {
       dlgResult = 0;
+      isHelpVisible = false;
+      showHelp = function(enable) {
+        isHelpVisible=enable;
+        var infos = document.getElementsByClassName("CLASS21113");
+        if (infos !== null) {
+          for (const info of infos) { info.style.display = enable ? "block" : "none"; }
+          dlgPopup.setWidth(enable ? 1020 : 600);
```

Comment:
Hmm, geht es wirklich nicht anders als mit einer festen verdrahtung dieser Werte (1020 vs. 600)? Das schreit danach das das ggf. irgendwelche Probleme bei gewissen Layouts oder in Zukunft verursachen könnte? Passt sichd as layout nicht automatisch an sobald man das `none` vs. `block` verwendet? Warum muss man hier feste Werte eingeben und woher kommen die?

Reply:
> Warum muss man hier feste Werte eingeben und woher kommen die?

Die 1020 waren schon immer fest drin. 

Und wenn die Hilfe nicht mehr da ist, ergibt es keinen Sinn, das Fenster mit den 2 Spalten so extrem breit zu lassen.
Da hab ich 600 genommen. 
Ziel war es ja gerade, es schmaler zu bekommen (mobile Gerätebildschirme).

Der Inhalt passt sich dynamisch ins Popup an.
Die Popup-Größe selbst jedoch nicht.
Deshalb die 600 und 1020.

Reply:
Das mit dem "block" und "none" war das was ich mir zum Ein-/Ausblenden von Elementen ergoogelt hatte

Reply:
> Und wenn die Hilfe nicht mehr da ist, ergibt es keinen Sinn, das Fenster mit den 2 Spalten so extrem breit zu lassen. Da hab ich 600 genommen. Ziel war es ja gerade, es schmaler zu bekommen (mobile Gerätebildschirme).
> 
> Der Inhalt passt sich dynamisch ins Popup an. Die Popup-Größe selbst jedoch nicht. Deshalb die 600 und 1020.

Ah ok, dachte das Popup kann sich bzgl. seiner Breite auch anpassen. Aber wo genau wird denn diese 1020 vorgegeben? Die 600 hast du ja nun so ausgewählt, aber das der ursprung 1020 ist? Wo genau ist das definiert?

Reply:
Alles klar. Danke für die Info. Dann passt das ja nun! Danke für den PR